src/translations/nbTranslations.json

```diff
@@ -85,16 +85,11 @@
       "show_full_nin": "Vis personnummer"
     },
     "users": {
-      "add_role_info": "Om du ikke finner brukeren, så må du sjekke at vedkommende er tilknyttet din institusjon og har logget på tjenesten tidligere.",
       "area_of_responsibility": "Ansvarsområde",
```

Comment:
Denne kan sikker også fjernes? evt flyttes inn under `editor`.

Siden for users slettes i denne PR, så ideelt sett burde hel `users` her også fjernes, hvor brukes de andre nøklene under her?